src/components/modals/ReferenceModal.vue

```diff
@@ -231,7 +231,7 @@ ul {
   }
   ul {
     li {
-      .icon { width: 6vh; }
+      .icon { width: 5.1vh; }
```

Comment:
Since this change is only useful on the maximized modal and there only below the `@media screen and (max-width: 991.98px)` breakpoint, could you please make it separate and target only the maximized modal?
```css
/** trim icon size on maximized one-column sheet **/
@media screen and (max-width: 991.98px) {
  .characters .modal.maximized ul li .icon {
    width: 5vh;
  }
}
```